### ISSUE TITLE
Roundup of Issues in Spreadsheet/Chart Components (Some bugs, some cosmetic, some feature requests) 

### ISSUE THREAD

**Issue by bobparks** [mit-cml/appinventor-sources#2785]
**Describe the bug**

Please see [attached sheet of questions](https://docs.google.com/document/d/123FlGRsg4SXXwV3CdNftYitNnKEy0rqDZihQsSFuwoE/edit), which describes the following questions and shows the blocks:

1.) Inconsistent way to reference columns
2.) Skip header in ImportfromSpreadsheet not working
3.) Question about reading non-adjacent columns
4.) Reading a spreadsheet where all the data is in row format instead of column format
5.) Wording on the ImportFromSpreadsheet method
6.) Layout of ChartData2D properties 

**Affects**

Various Spreadsheet and Chart blocks. 

- [ x] Designer
- [ x] Blocks editor
- [ ] Companion
- [ ] Compiled apps
- [ ] Buildserver
- [ ] Debugging
- [ ] Other... (please describe)

**Expected behavior**

**Steps to reproduce**

[Here](https://drive.google.com/file/d/1rJX_t54UlQT_T_lYMCTOL_AurxLrzC-4/view?usp=share_link) is an example AIA where some of these issues pop up.

**Comment by stezelMIT** [comment]
We also need the option of a Chart2D graph be able to start from (0,0).  The current system forces the y-axis to start from the lowest data point value and makes graph comparison impossible.

**Comment by bobparks** [comment]
> We also need the option of a Chart2D graph be able to start from (0,0). The current system forces the y-axis to start from the lowest data point value and makes graph comparison impossible.

@stezelMIT, @anqizzz8989 Good idea. I will file it as a separate issue with some images to support.
